--- srv/Program.cs	2020-01-20 10:11:43.977100947 +0300
+++ Program.cs	2020-01-20 10:21:00.184766593 +0300
@@ -18,6 +18,10 @@
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration((hostingContext, config) =>
+                {
+                    config.AddJsonFile("secrets.json", optional: true, reloadOnChange: true);
+                })
                 .ConfigureWebHostDefaults(webBuilder =>
                 {
                     webBuilder.UseStartup<Startup>();
